yaml/_dumper/dumper.ts

```diff
@@ -379,7 +379,7 @@ function foldString(string: string, width: number): string {
   // tslint:disable-next-line:no-conditional-assignment
   while ((match = lineRe.exec(string))) {
     const prefix = match[1],
-      line = match[2];
+      line = match[2]!;
```

Comment:
Can you please justify this change?

Reply:
I will change this with `match[2] || ""`, the check for line make sense inside the `foldLine` function